mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -1276,16 +1276,20 @@ class WidgetAdapter(
 
             // hide dividers before and after frame widgets
             val adapter = parent.adapter
-            val noDividerTypes = intArrayOf(TYPE_FRAME, TYPE_INVISIBLE)
             if (adapter != null) {
-                if (adapter.getItemViewType(position) in noDividerTypes) {
+                if (adapter.getItemViewType(position) == TYPE_FRAME) {
```

Comment:
I do not yet fully get the logic change. Before dividers were hidden before and after both frame and invisible widgets. Now the dividers are hidden
- before and after frames
- before invisible widgets
- after invisible widgets, but only if the next one is an invisible widget.

I don't get the last condition, can you explain that a bit? Shouldn't it be sufficient to remove dividers before invisible widgets, and the divider of the next widget will divide it from the previous one?